frontend/src/pages/Login.js

```diff
@@ -1,19 +1,22 @@
-import { useNavigate } from "react-router-dom";
-import { useLogin } from "../hooks/useLogin";
-
-const { useState } = require("react");
-
-const Login = () => {
-  const [email, setEmail] = useState("");
-  const [password, setPassword] = useState("");
-  const { login, error, isLoading, ok } = useLogin();
-  const navigate = useNavigate();
-
-  const handleSubmit = async (e) => {
-    e.preventDefault();
-    await login(email, password);
+import { useNavigate } from 'react-router-dom'
+import { useAuthContext } from '../hooks/useAuthContext'
+import { useLogin } from '../hooks/useLogin'
+const { useState } = require('react')
```

Comment:
If I'm not mistaken this again is formatting, not related to the next property adding (I reviewed commit by commit) !

Reply:
that is probably resolved @ctmbl, right? :yum: